data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -93,8 +93,19 @@ then
 
 	mkdir --parents /etc/kubernetes/static-pod-resources/etcd-member
 	cp tls/etcd-ca-bundle.crt /etc/kubernetes/static-pod-resources/etcd-member/ca.crt
-	cp --recursive etcd-bootstrap/bootstrap-manifests/secrets/etcd-all-serving /etc/kubernetes/static-pod-resources/etcd-member
-	cp --recursive etcd-bootstrap/bootstrap-manifests/secrets/etcd-all-peer /etc/kubernetes/static-pod-resources/etcd-member
+    # Temporarily ensure compatibility with both the new single cert secret and
```

Comment:
Use a tab so that it lines up with the rest of the block.

Reply:
Done